### DISCUSSION TITLE
How to batch import CSS style sheets.

### DISCUSSION THREAD

**Discussion by jackadam1981** [requarks/wiki#6654] [Help / Questions]
I am trying to import over 1000 Word documents and have already used the MultiDoc Converter for doc to HTML, as well as using the Git repository for synchronization and import.

I hope to maintain the original Word format content and see irregular CSS code in the converted HTML. However, it can only be imported in the page editing style for it to take effect.

I also found: Insert the JS and CSS under the Page Properties>Scripts/CSS dialog while editing the page You cannot add JS or CSS directly into the page content

Is there any simple way to introduce CSS styles to a page?

For example, automatically loaded CSS with the same name?

Or special CSS import tags?

In this way, I can write a script for automatic processing.

Sorry for my poor English, it's all machine translated.

**Comment by fnafox** [comment]
I've not used it myself but I'm pretty sure you could do this with GraphQL api. Check out the docs at https://docs.requarks.io/dev/api
PageMutation create allows you to enter CSS when creating a new page. You should be able to automate this process.

**Comment by jackadam1981** [comment]
The api is an option, but the manual only has Authentication, Examples, and no complete api description can be found.

Thanks for pointing me to a solution.

**Reply by P4mes4n** [reply]
The API is documented in the playground. Navigate to `/graphql` on your instance and you have the documentation on the right side. CSS can be updated via a page mutation.

**Comment by jackadam1981** [comment]
Thank you, because I am deploying on the enterprise network, I have not opened it. I changed the Internet computer to test, and it can be opened. Continue to study and use python scripts to automatically migrate css.
